src/modules/extendedCallWindow/i18n/cs_CZ.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
             [/^CAS$/]: [0, 1],
             [/^AP nebo AZ$/]: [2, 10],
             [/^VEA$/]: [3, 17],
-            [/^Technických automobilů$/]: [4],
+            [/^[Tt]echnický(ch) automobil(ů)$/]: [4],
```

Comment:
I think the ch and ů only appears in Plural? Then a `?` needs to be behind each `)` 😉